github.go

```diff
@@ -147,6 +147,37 @@ type GithubEvents []struct { // https://mholt.github.io/json-to-go/
 	} `json:"org,omitempty"`
 }
 
+type rawRateLimitSpecs struct {
+	Resources struct {
+		Core struct {
+			Limit     int
+			Remaining int
+			Reset     int64
+		}
+	}
+
+	Search struct {
```

Comment:
`Search` should be inside `Resources`: https://developer.github.com/v3/rate_limit/#response.
BTW, this could be removed as unneeded.